setup.py

```diff
@@ -46,6 +46,7 @@ def doc_version():
     extras_require={
         # this version requirement needs to be synchronized with the one in nmigen.back.verilog!
         "builtin-yosys": ["nmigen-yosys>=0.9.*"],
+        "remote-build": ["paramiko>=2.7.*"],
```

Comment:
This should be a semver constraint:
```suggestion
        "remote-build": ["paramiko~=2.7"],
```